Jenkinsfile

```diff
@@ -246,6 +246,59 @@ for(String envKeyName: context.env.keySet() as String[]){
         }
     }
 
+    // Backup (w/ pg_dump) before deploying
+    //if (("DEV".equalsIgnoreCase(stageDeployName) || isCD) {
+    if (isCI || isCD) {
+        _stage("Pre-deploy db backup - ${stageDeployName}", context){
+            node('master') {
+
+                String PROJECT = 'moe-gwells-test'
```

Comment:
Keep in mind it is running in a loop through all environments.  you can use `context.deployments[envKeyName]. projectName`. It will give you the appropriate project name (moe-gwells-dev, moe-gwells-test, moe-gwells-prod) for each environment (DEV, TEST, PROD)